.github/workflows/cypress.yml

```
@@ -0,0 +1,37 @@
+name: E2E test
+
+on:
+  push:
+    branches:
+      - develop
+      - trunk
+  pull_request:
+    branches:
+      - develop
+jobs:
+  cypress:
+    name: ${{ matrix.core.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        core:
+          - {name: 'WP latest', version: 'latest'}
+          - {name: 'WP minimum', version: 'WordPress/WordPress#5.7'}
+          - {name: 'WP trunk', version: 'WordPress/WordPress#master'}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Install dependencies
+      run: npm install
+    - name: Composer (optional)
+      run: composer install
+      continue-on-error: true
+    - name: Build (optional)
+      run: npm run build
+      continue-on-error: true
```

Required for asset build.

```suggestion
    - name: Build
      run: npm run build
```